sdk/metric/aggregation.go

```diff
@@ -95,6 +95,12 @@ type AggregationExplicitBucketHistogram struct {
 	// (-∞, 0], (0, 5.0], (5.0, 10.0], (10.0, 25.0], (25.0, 50.0],
 	// (50.0, 75.0], (75.0, 100.0], (100.0, 250.0], (250.0, 500.0],
 	// (500.0, 1000.0], (1000.0, +∞)
+	//
+	// The default bucket boundaries provided by DefaultAggregationSelector
+	// are optimized for millisecond measurements. When using a different unit
```

Comment:
It is not accurate to say these are "optimized for millisecond measurements". They are intended to evenly fit a variety of common measurements for many different units. There are many non-time based instruments that use histograms.